<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DADialogChartGuide</class>
 <widget class="QDialog" name="DADialogChartGuide">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chart Guide</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="listWidgetChartType">
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignHCenter</set>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="DA::DAChartAddCurveWidget" name="pageCurve"/>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DA::DAChartAddCurveWidget</class>
   <extends>QWidget</extends>
   <header>DAChartAddCurveWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DADialogChartGuide</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DADialogChartGuide</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
